bin/cipherstash-aws-cdk.ts

```diff
@@ -17,7 +17,13 @@ function getEnvVar(name: string): string {
 
 (async () => {
   const client = new sts.STSClient();
-  const command = new sts.GetCallerIdentityCommand({});
+
+  // Use the AWS_PROFILE environment variable to specify the profile to use.
+  // This is noted in the README, but it's worth mentioning here as well.
+  const command = new sts.GetCallerIdentityCommand({
+    profile: getEnvVar("AWS_PROFILE"),
```

Comment:
Was something breaking if you didn't specify the profile here? I think this will break when running ZeroKMS and CTS in separate accounts (and using separate AWS profiles) unless users remember to set `AWS_PROFILE` appropriately between CTS and ZeroKMS stack deploys.

It worked as intended while I was testing the other day (for both single-account and multi-account deployments), but I've been using `aws-vault` for authenticating commands.

The intent of the `GetCallerIdentity` call here is to look up the ARN of the caller so that the caller is allowed to manage KMS keys that CDK creates.